### ISSUE TITLE
Having trouble with how to start game api

### ISSUE THREAD

**Issue by TronB** [ga-wdi-boston/game-project#687]
I would like a 1:1 with someone to go over the game api, there are some gaps in my knowledge on how the api works and I think I may have to redo my game engine because of that.

**Comment by MicFin** [comment]
Can you add more to this?  You mentioned we cleared some things up.  Try to be as specific as possible and use examples.

**Comment by TronB** [comment]
There are two parts to this:
1) I tried to deploy my code to test my authentication before diving into my game api, but failed because it was aborted. I received a warning message:
```
Warning: Command failed: test -z "$(git status --porcelain)"  || (git status && false)
 Use --force to continue.

Aborted due to warnings.
```
I didn't fully understand how the api works so I thought it had to do with not having any curl scripts, which turned out to be not the case(through our discussion yesterday). I tried to google the warning, but still don't understand what the issue is.

2) The other issue I have is I think I may have overlapped some of the functionality meant for the game api into my game engine. To be more specific I am referring to the requirements:

POST for starting a new game
PATCH for making a move during a game

I used conditionals and click events to implement those functionalities. I had thought at the time that it was what would be needed to make the game work. I'm worried that I'll have to re-write a lot of the game engine to write out the api and am struggling on where to start.
